style-editor.scss

```diff
@@ -82,6 +82,7 @@ body {
 		width: calc(6 * (100vw / 12 ));
 	}
 
+	// Only the top level blocks need specific widths, therefore override for every nested block.
```

Comment:
See also https://github.com/WordPress/twentynineteen/pull/526/files#r232209089 — I wish there was a different way we could write this rule.

Right now we are setting widths on every `wp-block`, and then _unsetting them again_ on every nested block. It would be more solid if we could target only the top level `.wp-block`, because I could easily imagine a Gutenblock that is not meant to be 100% wide.